worlds/kh1/Options.py

```diff
@@ -155,6 +155,25 @@ class FinalRestDoor(Choice):
     option_postcards = 2
     option_superbosses = 3
 
+class RequiredPostcards(Range):
+    """
+    If your goal is set to "Postcards", defines how many postcards are needed to achieve victory.
+    """
+    display_name = "Required Postcards"
+    default = 10
+    range_start = 1
+    range_end = 10
+
+class RequiredPuppies(Range):
+    """
+    If your goal is set to "Puppies", defines how many puppies are needed to achieve victory.
+    Note: The required puppies will round down to the nearest amount of puppies that is a location.
```

Comment:
```suggestion
    Note: The required number of puppies will round down to the nearest amount of puppies that are given from a location.
```

Reply:
There may be some better way to say this - but every 10 puppies returned are a location.  So what I am trying to say by this is, if the player inputs that 37 puppies are required for their victory, the code will round that down to 30.  Only exception to this is 99 puppies returned, which is also a location and has a special condition to bypass the rounding if the player inputs 99.

My wording may not be clear.

Reply:
I wonder if this should just be a choice then instead of a range

Reply:
Yeah that's a better idea!  Changed!